jobs/opensearch/spec

```diff
@@ -28,6 +28,10 @@ templates:
   config/plugin-stats-metadata: config/opensearch-performance-analyzer/plugin-stats-metadata
   config/opensearch-security/config.yml.erb: config/opensearch-security/config.yml
   config/opensearch-security/roles_mapping.yml.erb: config/opensearch-security/roles_mapping.yml
+  config/opensearch-notifications/notifications.erb : config/opensearch-notifications/notifications.yml
```

Comment:
Do these config files existing in the package files on the VM? If so, should we copy those default files? 

I don't know what the defaults are for these files and if we want them